packages/backend/src/managers/catalogManager.ts

```diff
@@ -66,7 +66,7 @@ export class CatalogManager {
     try {
       this.watchCatalogFile(catalogPath); // do not await, we want to do this async
     } catch (err: unknown) {
-      console.error("unable to watch catalog file, changes to the catalog file won't be reflected to the UI", err);
+      console.error(`unable to watch catalog file, changes to the catalog file won't be reflected to the UI`, err);
```

Comment:
Not a template ?

Reply:
else we need to replace won't by will not